opm/autodiff/BlackoilSolventModel.hpp

```diff
@@ -198,12 +223,19 @@ namespace Opm {
         computeMassFlux(const int               actph ,
                         const V&                transi,
                         const ADB&              kr    ,
+                        const ADB&              mu    ,
+                        const ADB&              rho   ,
                         const ADB&              p     ,
                         const SolutionState&    state );
 
         const std::vector<PhasePresence>
         phaseCondition() const {return this->phaseCondition_;}
 
+        // compute effective viscosities (mu_eff_) and effective b factors (b_eff_)  using the ToddLongstaff model
+        void calculateEffectiveProperties(const SolutionState&  state);
+
+        // compute density and viscosity using the ToddLongstaff mixing model
+        void ToddLongstaffModel(std::vector<ADB>& viscosity, std::vector<ADB>& density, const std::vector<ADB>& saturations, const Opm::PhaseUsage pu);
```

Comment:
Method names should not start with a capital letter (reserved for class or namespace names an enums).
Perhaps computeToddLongstaffMixing() or something?